### ISSUE TITLE
Plugin not found by gradle

### ISSUE THREAD

**Issue by patrickjonesuk** [dhakehurst/kotlin.gradle.plugin.exportPublic#1]
Following the instructions on the [plugin page](https://plugins.gradle.org/plugin/net.akehurst.kotlin.gradle.plugin.exportPublic), I have attempted to add the plugin by both methods:
```kt
plugins {
  id("net.akehurst.kotlin.gradle.plugin.exportPublic") version "1.1.0"
}
```
and
```kt
buildscript {
  repositories {
    maven {
      url = uri("https://plugins.gradle.org/m2/")
    }
  }
  dependencies {
    classpath("net.akehurst.kotlin.gradle.plugin:exportPublic:1.1.0")
  }
}

apply(plugin = "net.akehurst.kotlin.gradle.plugin.exportPublic")
```
Both the methods produce the same error during `build` task:
```
Execution failed for task ':compileCommonMainKotlinMetadata'.
> Error while evaluating property 'filteredArgumentsMap' of task ':compileCommonMainKotlinMetadata'
   > Could not resolve all files for configuration ':kotlinCompilerPluginClasspathMetadataCommonMain'.
      > Could not find net.akehurst.kotlin.gradle.plugin:exportPublic:1.1.0.
        Searched in the following locations:
          - https://repo.maven.apache.org/maven2/net/akehurst/kotlin/gradle/plugin/exportPublic/1.1.0/exportPublic-1.1.0.pom
        If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
        Required by:
            project :
```
By adding `maven("https://plugins.gradle.org/m2/")` to the top-level `dependencies` block (or in `pluginManagement.repositories` as it should be), and declaring the plugin in the `plugins` block,
the same error occurs, but an additional URL is added to the "Searched in the following locations:" section: `https://plugins.gradle.org/m2/net/akehurst/kotlin/gradle/plugin/exportPublic-native/1.0.0/exportPublic-native-1.1.0.pom`

**Comment by dhakehurst** [comment]
try the latest release,
gradle 7.4.2
kotlin 1.6.21
exportPublic 1.6.21

**Comment by patrickjonesuk** [comment]
That issue seems to have been resolved with exportPublic `1.6.21`, and it works as intended when only js and jvm targets are configured. However, adding a native target, such as `linuxX64` or `mingwX64`, causes an error during build:
```
* What went wrong:
Execution failed for task ':compileKotlinLinux'.
> Could not resolve all files for configuration ':kotlinCompilerPluginClasspathLinuxMain'.
   > Could not find net.akehurst.kotlin.gradle.plugin:exportPublic-native:1.6.21.
     Searched in the following locations:
       - file:/home/pjones/.m2/repository/net/akehurst/kotlin/gradle/plugin/exportPublic-native/1.6.21/exportPublic-native-1.6.21.pom
       - https://repo.maven.apache.org/maven2/net/akehurst/kotlin/gradle/plugin/exportPublic-native/1.6.21/exportPublic-native-1.6.21.pom
       - https://plugins.gradle.org/m2/net/akehurst/kotlin/gradle/plugin/exportPublic-native/1.6.21/exportPublic-native-1.6.21.pom
     Required by:
         project :
```
This appears to be looking in the repositories defined in the top-level `repositories` block for `exportPublic-native`. I have declared the plugin like this:
```kt
plugins {
    id("net.akehurst.kotlin.gradle.plugin.exportPublic") version("1.6.21")
}
```

**Comment by patrickjonesuk** [comment]
Solved by adding `kotlin.native.useEmbeddableCompilerJar=true` to `gradle.properties`. Thanks for the help.
